.testcoverage.yml

```diff
@@ -63,8 +63,12 @@ override:
     path: go/interpreter/lfvm/ct.go
   - threshold: 38
     path: go/interpreter/lfvm/lfvm.go
-  - threshold: 0
+  - threshold: 100
+    path: go/interpreter/lfvm/converter.go
+  - threshold: 100
     path: go/interpreter/lfvm/memory.go
+  - threshold: 100
+    path: go/interpreter/lfvm/interpreter.go
```

Comment:
package coverage should be 100, and here only specify the files that have a lower coverage. wdyt ?

Reply:
ok

Reply:
I tried that, it seems to be that overrides are exclusive, and I cannot set the lfvm to 100 yet. 
Hopefully we can change this soon to 100% for every file.